backend/src/pong/game_invite/game-invite.controller.ts

```diff
@@ -0,0 +1,36 @@
+import {
+	Controller,
+	Get,
+	Post,
+	Body,
+	Patch,
+	Param,
+	Delete,
+} from '@nestjs/common';
+import { GameInvitesService } from './game-invite.service';
+import { CreateGameInviteDto } from './dto/create-game-invite.dto';
+
+@Controller('game-invites')
+export class GameInvitesController {
+	constructor(private readonly gameInvitesService: GameInvitesService) {}
+
+	@Post()
+	create(@Body() createGameInviteDto: CreateGameInviteDto) {
+		return this.gameInvitesService.create(createGameInviteDto);
+	}
+
+	@Get()
+	findAll() {
+		return this.gameInvitesService.findAll();
+	}
+
+	@Get(':id')
+	findOne(@Param('id') id: number) {
+		return this.gameInvitesService.findOne({ where: { id } });
+	}
+
```

Comment:
probably on purpose, but is it correct that there is no update functionality? 
as in we can't update game-invite? doe sort of makes sense(?)

Reply:
Exactly, there isn't any info to update as the users stay constant, until the invite is removed :)